src/app/vm/vm-creation/vm-creation.component.html

```diff
@@ -103,24 +103,12 @@ <h4>{{ 'VM_PAGE.VM_CREATION.SERVICE_OFFERING' | translate }}</h4>
 
           <div *ngIf="diskOfferingsAreAllowed">
             <h4>{{ 'VM_PAGE.VM_CREATION.DISK_OFFERING' | translate }}</h4>
-            <mat-form-field
-              class="full-width-input"
-              *loading="diskOfferingsAreLoading"
-            >
-              <mat-select
-                class="form-select-control"
-                name="diskOfferingSelector"
-                [ngModel]="vmCreationState.diskOffering"
-                (selectionChange)="diskOfferingChange.emit($event.value)"
-              >
-                <mat-option
-                  *ngFor="let offering of diskOfferings"
-                  [value]="offering"
-                >
-                  {{ offering.name }}
-                </mat-option>
-              </mat-select>
-            </mat-form-field>
+            <cs-disk-offering-selector
+              name="diskOffering"
+              [diskOfferings]="diskOfferings"
+              [diskOfferingId]="vmCreationState.diskOffering?.id"
+              (change)="diskOfferingChange.emit($event)"
+            ></cs-disk-offering-selector>
```

Comment:
If you use the implementation of ControlValueAccessor, you are able to set input value through ngModel. Or it is unnecessary, if you prefer to set input value as usual